docs/tutorials/postgres-dlp.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 9
+slug: /tutorials/postgres-dlp
+---
+
+# Data Loss Prevention (DLP) using Postgesql
+
+In this tutorial we will show how to connected with any database service from any private networking. You only need to install the hoop agent to get access and apply DLP by default.
```

Comment:
You could add a section with pre-requisites for running this tutorial, example:

- Hoop command line (Point to https://hoop.dev/docs/installing/command-line)
- Docker installed in the remote host

Reply:
Yeah, the section pre-requisites is nice, we could improve the title about the Hoop agent.
What's the diference between these links, I think it's the same.

- https://hoop.dev/docs/quickstarts
- https://hoop.dev/docs/installing/command-line